### DISCUSSION TITLE
Vercel for one-click deployment

### DISCUSSION THREAD

**Discussion by Nanzhexi** [open-webui/open-webui#2486] [General]
**Is your feature request related to a problem? Please describe.**
We need to use Vercel for one-click deployment because deploying through Docker is too cumbersome.

**Comment by nickfixit** [comment]
I'm trying to do a similar thing. Basically it needs a way to deploy the front end via vercel. And you should then have it connect to the backend if you need local llama or some other support.

**Reply by nickfixit** [reply]
Any advice from anyone on how to proceed. I'll continue with an autogpt/crew to try and set one up.

**Reply by arthrod** [reply]
How did you solve this issue?

**Reply by solomonshalom** [reply]
Hmm! I think few folks would much rather do it via Vercel. Is there any update on this?

**Comment by arthrod** [comment]
Render: https://dashboard.render.com/blueprint/new?repo=https://github.com/open-webui/open-webui
Repocloud: https://repocloud.io/details/?app_id=271

**Reply by solomonshalom** [reply]
Woah! This is pretty cool!

Any chance w/ Vercel doe?

**Reply by arthrod** [reply]
I doesn't submit myself these... What's the reason you prefer vercel (curious)?

**Reply by solomonshalom** [reply]
> I doesn't submit myself these... What's the reason you prefer vercel (curious)?

I see!

Sure! I prefered Vercel because of it's great developer experience + it's pretty snappy (ALSO! I am just very used to it, loll 😅 )

But Render seems really cool, I might just use that 🤔

**Reply by bernaferrari** [reply]
Render is missing render.yaml file.

I like vercel because I can deploy for free. I want to deploy web ui for 5 people only.

**Reply by dakipro** [reply]
Any way of adding the missing render.yaml file?
I am new to self hosting, have no idea how to make this work easily/properly

**Reply by ghost** [reply]
> Render is missing render.yaml file.
> 
> I like vercel because I can deploy for free. I want to deploy web ui for 5 people only.

hello, how did you deploy to vercel? please tell me, thanks

**Reply by myworldbox** [reply]
Plz do a vercel / github deployment workflow, vercel is the best deployment tool.

**Comment by GameCodeMode** [comment]
any update?
